CMakeLists.txt

```diff
@@ -1475,19 +1462,17 @@ else()
 endif()
 add_dependencies(mixxx-lib mixxx-res mixxx-script)
 
+file(READ src/_version.h MIXXX_VERSION_FILECONTENT)
+string(REGEX REPLACE "^.*#define MIXXX_VERSION \"(.*)\".*$" "\\1" MIXXX_VERSION "${MIXXX_VERSION_FILECONTENT}")
+
 # Windows-only resource file
 if(WIN32)
-  string(TIMESTAMP MIXXX_YEAR "%Y")
-
-  file(READ src/_version.h MIXXX_VERSION_FILECONTENT)
-  string(REGEX REPLACE "^.*#define MIXXX_VERSION \"(.*)\".*$" "\\1" MIXXX_VERSION "${MIXXX_VERSION_FILECONTENT}")
-
   # Remove anything after ~ or - in the version number and replace the dots with commas
   string(REGEX REPLACE "^([^~-]+).*$" "\\1" MIXXX_FILEVERSION "${MIXXX_VERSION}")
   string(REPLACE "." "," MIXXX_FILEVERSION "${MIXXX_FILEVERSION}")
-  if(NOT GIT_COMMIT_COUNT STREQUAL "unknown")
-    set(MIXXX_FILEVERSION "${MIXXX_FILEVERSION},${GIT_COMMIT_COUNT}")
-    string(REPLACE "+" "" MIXXX_FILEVERSION "${MIXXX_FILEVERSION}")
+  if(NOT GIT_COMMIT_DESCRIPTION STREQUAL "unknown-commit")
+    string(REGEX MATCH "-([0-9]*)-" GIT_COMMITS_SINCE_LAST_TAG "${GIT_COMMIT_DESCRIPTION}")
+    set(MIXXX_FILEVERSION "${MIXXX_FILEVERSION},${CMAKE_MATCH_1}")
```

Comment:
string(FIND "${MIXXX_VERSION}" "pre" MIXXX_PRERELEASE_POSITION)

below requires also an update

string(FIND "${MIXXX_VERSION}" "pre" MIXXX_PRERELEASE_POSITION)

Reply:
The -modified string needs to become the VS_FF_PATCHED flag 

https://docs.microsoft.com/en-us/windows/win32/menurc/versioninfo-resource

Reply:
> I think we need to check for "alpha" and "beta", because both will look else look like the release version e.g. 2,3,0,23

I think we should use VS_FF_PRERELEASE for that.